mysql/datadog_checks/mysql/config.py

```diff
@@ -30,6 +30,10 @@ def __init__(self, instance):
         self.charset = instance.get('charset')
         self.deep_database_monitoring = is_affirmative(instance.get('deep_database_monitoring', False))
         self.statement_metrics_limits = instance.get('statement_metrics_limits', None)
+        self.full_statement_text_cache_max_size = instance.get('full_statement_text_cache_max_size', 10000)
+        self.full_statement_text_samples_per_hour_per_query = instance.get(
```

Comment:
This variable seems inverted. Shouldn't we just specify the cache time?

Reply:
I chose "samples per hour per query" as the unit of specifying the rate limit as I think it's a bit easier to reason about than the raw TTL. That's the way all of the other rate limits work: `explained_statements_per_hour_per_query`, `samples_per_hour_per_query`.